spec/helpers/simple_server_env_helper_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'rails_helper'
+
+RSpec.describe SimpleServerEnvHelper do
+  include SimpleServerEnvHelper
+
+  let(:simple_server_env) {"SIMPLE_SERVER_ENV"}
```

Comment:
Just a styling thing: put spaces after opening and before closing curly brace. You can set this up in RubyMine to do automatically.

Reply:
Implemented changes.